### ISSUE TITLE
Incorrect documentation on log_loss

### ISSUE THREAD

**Issue by nightrome** [tensorflow/tensorflow#7215]
1) I noticed that [1] states that tf.contrib.losses.log_loss is deprecated and that tf.losses.log_loss should be used instead. However in v0.12 that function does not exist and the documentation in [2] does not talk about this. So which one is correct?
2) Furthermore it seems that tf.contrib.losses.log_loss only returns a scalar for the summed loss of all dimensions. Is there a function more like tf.nn.softmax_cross_entropy_with_logits (but without the softmax), that returns the loss for each element separately?

[1] https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/api_docs/python/functions_and_classes/shard6/tf.contrib.losses.log_loss.md
[2] https://www.tensorflow.org/api_docs/python/contrib.losses/other_functions_and_classes#log_loss

**Comment by girving** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. In particular, we do not intend the documentation at `master` to describe the differences with previous versions of TensorFlow `contrib.  Thanks!
